python/src/WeibullMax_doc.i.in

```diff
@@ -5,14 +5,20 @@ Parameters
 ----------
 beta : float
     Scale parameter :math:`\beta > 0`.
+
+    Default value is 1.0.
 alpha : float
     Shape parameter :math:`\alpha > 0`.
+
+    Default value is 1.0.
 gamma : float, optional
     Location parameter.
 
+    Default value is 0.0.
+
 Notes
 -----
-Its cumulative and probability density functions are defined as:
+Its cumulative density function and probability distribution function are defined as:
```

Comment:
I believe that this is the other way around:
```suggestion
Its cumulative distribution function and probability density function are defined as:
```

Reply:
ok